toolchain/check/testdata/function/builtin/no_prelude/import.carbon

```diff
@@ -77,101 +77,101 @@ var arr: [i32; Core.AsIntLiteral(Core.TestAdd(Core.AsI32(1), Core.AsI32(2)))] =
 // CHECK:STDOUT:   %AsI32.decl: %AsI32.type = fn_decl @AsI32 [template = constants.%AsI32] {
 // CHECK:STDOUT:     %a.patt: Core.IntLiteral = binding_pattern a
 // CHECK:STDOUT:     %a.param_patt: Core.IntLiteral = value_param_pattern %a.patt, runtime_param0
-// CHECK:STDOUT:     %return.patt: %i32 = return_slot_pattern
-// CHECK:STDOUT:     %return.param_patt: %i32 = out_param_pattern %return.patt, runtime_param1
+// CHECK:STDOUT:     %return.patt: %i32.builtin = return_slot_pattern
```

Comment:
Why do all these uses of `i32` refer to the builtin, rather than to `Core.Int(32)`?